import{s as V,o as z,f as g,a as $,e as G,g as _,h as b,d as p,c as v,j as A,i as h,I as W,K as k,n as B,M as H,l as U,m as q,p as O}from"../chunks/scheduler.f368f94a.js";import{S as L,i as K,b as y,d as w,m as S,a as u,t as f,e as x,g as N,c as R}from"../chunks/index.b73477ac.js";import{A as D,e as j}from"../chunks/index.522fdf1d.js";import{f as C}from"../chunks/app.438d6912.js";import{b as F}from"../chunks/paths.e9ed168c.js";import{i as Q}from"../chunks/experience.b6fa608a.js";import{i as J}from"../chunks/projects.e0552c92.js";import{g as P,i as X}from"../chunks/skills.950a306a.js";import{S as Y}from"../chunks/SearchPage.8859c55a.js";import{C as Z}from"../chunks/Chip.146c7440.js";import{U as I}from"../chunks/UIcon.46d026ca.js";const ee="Search",te=[{slug:"slick-portfolio-research-sunstamp",color:"#007acc",description:"We propose a supervised Non-negative Matrix Factorization (NMF) framework designed to integrate spatial transcriptomics with complementary molecular or imaging modalities. The project addresses a key gap in spatial omics by enabling joint modeling of gene expression and pathology-derived molecular features. It demonstrated strong predictive performance (AUC >= 0.90) in Alzheimer’s disease and high correlation with myocardial injury markers, revealing biologically interpretable spatial gene modules.",shortDescription:"Supervised Factorization to Associate Spatial Transcriptomics with Complementary Molecular Readouts",links:[{to:"https://www.biorxiv.org/content/10.1101/2025.09.28.679034v1.abstract",label:"bioRxiv"},{to:"https://github.com/f12-mou/SuNSTAMP",label:"Github"}],logo:D.Python,name:"SuNSTAMP",period:{from:new Date(2024,2,1),to:new Date(2025,3,25)},skills:P("python","numpy","pandas"),type:"Undergraduate Thesis (Research) Project"},{slug:"slick-portfolio-research-spatialomics-toolkit",color:"#4a90e2",description:"Worked with Dr. Md. Abul Hassan Samee to design and develop a spatial transcriptomics tutorial demonstrating analysis pipelines for MERFISH and Visium 2.1.0 datasets using the spatialdata toolkit. The tutorial established reproducible workflows for the bioinformatics research community and addressed the challenge of standardizing analyses across heterogeneous technologies. The pipeline was later used in live demonstrations at the “Workshop: Mastering Spatial Data Analysis: From Basics to Cutting-Edge Innovations” as part of the Festival of Genomics and Biodata Boston 2025.",shortDescription:"Contributed to the development of a spatial transcriptomics tutorial and unified analysis pipeline using the spatialdata toolkit under Dr. Md. Abul Hassan Samee.",links:[{to:"https://github.com/f12-mou/spatialomics-toolkit",label:"GitHub"},{to:"https://festivalofgenomics.com/boston/2025-agenda/mastering-spatial-data-analysis-basics-cutting-edge-innovations",label:"Workshop Details"}],logo:D.Python,name:"spatialomics-toolkit",period:{from:new Date(2025,5,15),to:new Date(2025,9,15)},skills:P("python","scikit-learn","matplotlib"),type:"Remote Research Project"},{slug:"slick-portfolio-cell",color:"#4a90e2",description:"Worked with Dr. Md. Abul Hassan Samee to model spatially resolved cell–cell communication networks in axolotl regeneration datasets. Designed neighborhood-based matrices and heuristic strategies to efficiently infer multi-step ligand–receptor signaling cascades. Identified spatially coherent signaling chains and candidate hubs, providing insights into regeneration-associated pathways.",shortDescription:"Modeled spatially resolved cell–cell communication in axolotl regeneration datasets under Dr. Md. Abul Hassan Samee.",links:[],logo:D.Python,name:"Cell–Cell Communication Network Using Spatial Ligand–Receptor Correlation",period:{from:new Date(2025,7,15),to:void 0},skills:P("python","scikit-learn","matplotlib"),type:"Ongoing Research Project"},{slug:"slick-portfolio-ngcf",color:"#4a90e2",description:"Working with Dr. Mohammad Saifur Rahman to develop an advanced recommendation framework by enhancing Neural Graph Collaborative Filtering(NGCF) with neighborhood - based contrastive learning and graph attention mechanisms. Evaluated on Gowalla and Amazon Books datasets, achieving improved performance in Normalized Discounted Cumulative Gain(NDCG@k) and Recall@k metrics.",shortDescription:"Working with Dr. Mohammad Saifur Rahman to develop an advanced recommendation framework by enhancing Neural Graph Collaborative Filtering(NGCF) with neighborhood - based contrastive learning and graph attention mechanisms.",links:[],logo:D.Python,name:"Graph-Based Contrastive Recommendation System",period:{from:new Date(2024,11,15),to:void 0},skills:P("python","scikit-learn","matplotlib"),type:"Ongoing Research Project | Targeted Venue: NeurIPS Workshop"}];function E(c,a,i){const e=c.slice();return e[4]=a[i],e}function oe(c){let a,i,e,o;const r=[ie,ne],n=[];function s(t,m){return t[1].length===0?0:1}return i=s(c),e=n[i]=r[i](c),{c(){a=g("div"),e.c()},l(t){a=_(t,"DIV",{});var m=b(a);e.l(m),m.forEach(p)},m(t,m){h(t,a,m),n[i].m(a,null),o=!0},p(t,m){let l=i;i=s(t),i===l?n[i].p(t,m):(N(),f(n[l],1,1,()=>{n[l]=null}),R(),e=n[i],e?e.p(t,m):(e=n[i]=r[i](t),e.c()),u(e,1),e.m(a,null))},i(t){o||(u(e),o=!0)},o(t){f(e),o=!1},d(t){t&&p(a),n[i].d()}}}function ae(c){let a,i,e,o,r="Try typing something...",n;return i=new I({props:{icon:"i-carbon-search-locate-mirror",classes:"text-2em"}}),{c(){a=g("div"),y(i.$$.fragment),e=$(),o=g("span"),o.textContent=r,this.h()},l(s){a=_(s,"DIV",{class:!0});var t=b(a);w(i.$$.fragment,t),e=v(t),o=_(t,"SPAN",{["data-svelte-h"]:!0}),W(o)!=="svelte-16h9bak"&&(o.textContent=r),t.forEach(p),this.h()},h(){A(a,"class","flex-1 self-center col-center m-t-10 gap-5 font-300 text-[var(--accent-text)]")},m(s,t){h(s,a,t),S(i,a,null),k(a,e),k(a,o),n=!0},p:B,i(s){n||(u(i.$$.fragment,s),n=!0)},o(s){f(i.$$.fragment,s),n=!1},d(s){s&&p(a),x(i)}}}function ne(c){let a,i,e=j(c[1]),o=[];for(let n=0;n<e.length;n+=1)o[n]=T(E(c,e,n));const r=n=>f(o[n],1,1,()=>{o[n]=null});return{c(){a=g("div");for(let n=0;n<o.length;n+=1)o[n].c();this.h()},l(n){a=_(n,"DIV",{class:!0});var s=b(a);for(let t=0;t<o.length;t+=1)o[t].l(s);s.forEach(p),this.h()},h(){A(a,"class","flex flex-row flex-wrap gap-1")},m(n,s){h(n,a,s);for(let t=0;t<o.length;t+=1)o[t]&&o[t].m(a,null);i=!0},p(n,s){if(s&2){e=j(n[1]);let t;for(t=0;t<e.length;t+=1){const m=E(n,e,t);o[t]?(o[t].p(m,s),u(o[t],1)):(o[t]=T(m),o[t].c(),u(o[t],1),o[t].m(a,null))}for(N(),t=e.length;t<o.length;t+=1)r(t);R()}},i(n){if(!i){for(let s=0;s<e.length;s+=1)u(o[s]);i=!0}},o(n){o=o.filter(Boolean);for(let s=0;s<o.length;s+=1)f(o[s]);i=!1},d(n){n&&p(a),H(o,n)}}}function ie(c){let a,i,e,o,r="Oops ! nothing to show !",n;return i=new I({props:{icon:"i-carbon-cube",classes:"text-2em"}}),{c(){a=g("div"),y(i.$$.fragment),e=$(),o=g("span"),o.textContent=r,this.h()},l(s){a=_(s,"DIV",{class:!0});var t=b(a);w(i.$$.fragment,t),e=v(t),o=_(t,"SPAN",{["data-svelte-h"]:!0}),W(o)!=="svelte-1peshqu"&&(o.textContent=r),t.forEach(p),this.h()},h(){A(a,"class","flex-1 self-center col-center m-t-10 gap-5 font-300 text-[var(--accent-text)]")},m(s,t){h(s,a,t),S(i,a,null),k(a,e),k(a,o),n=!0},p:B,i(s){n||(u(i.$$.fragment,s),n=!0)},o(s){f(i.$$.fragment,s),n=!1},d(s){s&&p(a),x(i)}}}function se(c){let a,i,e,o=c[4].name+"",r,n,s;return a=new I({props:{icon:c[4].icon}}),{c(){y(a.$$.fragment),i=$(),e=g("span"),r=U(o),n=$()},l(t){w(a.$$.fragment,t),i=v(t),e=_(t,"SPAN",{});var m=b(e);r=q(m,o),m.forEach(p),n=v(t)},m(t,m){S(a,t,m),h(t,i,m),h(t,e,m),k(e,r),h(t,n,m),s=!0},p(t,m){const l={};m&2&&(l.icon=t[4].icon),a.$set(l),(!s||m&2)&&o!==(o=t[4].name+"")&&O(r,o)},i(t){s||(u(a.$$.fragment,t),s=!0)},o(t){f(a.$$.fragment,t),s=!1},d(t){t&&(p(i),p(e),p(n)),x(a,t)}}}function T(c){let a,i;return a=new Z({props:{href:`${F}/${c[4].to}`,classes:"flex flex-row items-center gap-2",$$slots:{default:[se]},$$scope:{ctx:c}}}),{c(){y(a.$$.fragment)},l(e){w(a.$$.fragment,e)},m(e,o){S(a,e,o),i=!0},p(e,o){const r={};o&2&&(r.href=`${F}/${e[4].to}`),o&130&&(r.$$scope={dirty:o,ctx:e}),a.$set(r)},i(e){i||(u(a.$$.fragment,e),i=!0)},o(e){f(a.$$.fragment,e),i=!1},d(e){x(a,e)}}}function re(c){let a,i,e,o,r,n;const s=[ae,oe],t=[];function m(l,d){return l[0]?1:0}return e=m(c),o=t[e]=s[e](c),{c(){a=g("div"),i=$(),o.c(),r=G(),this.h()},l(l){a=_(l,"DIV",{class:!0}),b(a).forEach(p),i=v(l),o.l(l),r=G(),this.h()},h(){A(a,"class","flex flex-col items-stretch gap-10 p-2")},m(l,d){h(l,a,d),h(l,i,d),t[e].m(l,d),h(l,r,d),n=!0},p(l,d){let M=e;e=m(l),e===M?t[e].p(l,d):(N(),f(t[M],1,1,()=>{t[M]=null}),R(),o=t[e],o?o.p(l,d):(o=t[e]=s[e](l),o.c()),u(o,1),o.m(r.parentNode,r))},i(l){n||(u(o),n=!0)},o(l){f(o),n=!1},d(l){l&&(p(a),p(i),p(r)),t[e].d(l)}}}function le(c){let a,i;return a=new Y({props:{title:ee,$$slots:{default:[re]},$$scope:{ctx:c}}}),a.$on("search",c[2]),{c(){y(a.$$.fragment)},l(e){w(a.$$.fragment,e)},m(e,o){S(a,e,o),i=!0},p(e,[o]){const r={};o&131&&(r.$$scope={dirty:o,ctx:e}),a.$set(r)},i(e){i||(u(a.$$.fragment,e),i=!0)},o(e){f(a.$$.fragment,e),i=!1},d(e){x(a,e)}}}function ce(c,a,i){let e="",o=[];z(()=>{let n=new URLSearchParams(window.location.search);i(0,e=n.get("q")??"")});const r=n=>i(0,e=n.detail.search);return c.$$.update=()=>{c.$$.dirty&3&&(i(1,o=[]),o.push(...C(J,e).map(n=>({data:n,icon:"i-carbon-cube",name:n.name,to:`projects/${n.slug}`}))),o.push(...C(te,e).map(n=>({data:n,icon:"i-carbon-cube",name:n.name,to:`rprojects/${n.slug}`}))),o.push(...C(X,e).map(n=>({data:n,icon:"i-carbon-software-resource-cluster",name:n.name,to:`skills/${n.slug}`}))),o.push(...C(Q,e).map(n=>({data:n,icon:"i-carbon-development",name:`${n.name} @ ${n.company}`,to:`experience/${n.slug}`}))))},[e,o,r]}class ke extends L{constructor(a){super(),K(this,a,ce,le,V,{})}}export{ke as component};
